packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/ios/integrationsteps.md

```diff
@@ -53,15 +47,14 @@ and returns either a fatal error or success with a token.
 }
 ```
 
-#### Step 3: Send user to home screen after successful sign-in
+### Step 3: Send user to home screen after successful sign in
```

Comment:
Sorry, I didn't edit the contents of this page, I just added the get user info part at the bottom. You're right though: we use "sign-in" (hyphen) as an adjective. Susan mentioned that our style is to avoid using "sign in" as a noun. I'll update this heading, and leave the other content/heading changes for the reorg PRs.